ocs_ci/helpers/network_policy_helpers.py

```
@@ -0,0 +1,400 @@
+
+    Raises:
+        AssertionError: If no CSV matches the prefix.
+    """
+    namespace = namespace or config.ENV_DATA["cluster_namespace"]
+    ocp_obj = OCP(kind="ClusterServiceVersion", namespace=namespace)
+    csvs = ocp_obj.get().get("items", [])
+    for csv in csvs:
+        name = csv["metadata"]["name"]
+        if name.startswith(csv_prefix):
+            return name
+    raise AssertionError(f"No CSV found with prefix '{csv_prefix}' in {namespace}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)